lib/router/router.js

```diff
@@ -177,8 +177,8 @@ export default class Router extends EventEmitter {
     const routeInfo = {}
 
     try {
-      const { Component, err, xhr } = routeInfo.data = await this.fetchComponent(route)
-      const ctx = { err, xhr, pathname, query }
+      const { Component, err } = routeInfo.data = await this.fetchComponent(route)
+      const ctx = { err, pathname, query }
```

Comment:
The reason is the prefetched version doesn't have a `xhr` object. It'll see more about this.